templates/learn.html

```diff
@@ -68,5 +68,18 @@
       </div>
     </div>
   </a>
+  <a class="card" href="/faq">
+    <div class="card-image">
+      <img src="/assets/bevy_icon_dark.svg" class="centered-card-image" alt="Bevy logo" />
```

Comment:
we might want a different icon here at some point, but for now it's fine.

Reply:
Yeah we need unique icons for a number of items in this list.
(Bevy Assets, Bevy Web Examples, and now FAQ)